alluxio-ui/master/src/containers/pages/Browse/Browse.tsx

```diff
@@ -195,40 +196,74 @@ export class Browse extends React.Component<AllProps, IBrowseState> {
                    onChange={pathInputHandler}/>
           </FormGroup>
           <FormGroup className="mb-2 mr-sm-2">
-            <Button tag={Link} to={`/browse?path=${path}${queryStringSuffix}`} color="primary"
+            <Button tag={Link} to={`/browse?path=${path}${queryStringSuffix}`} color="secondary"
                     disabled={path === lastFetched.path}>Go</Button>
           </FormGroup>
         </Form>
         <Table hover={true}>
           <thead>
           <tr>
-            <th/>
+            <th></th>
```

Comment:
Could you comment that this is to leave an icon placeholder? Also is the shorthand no longer preferred?

Reply:
Great catch, the linter settings changed when we removed react-scripts-ts, I think about adding explicit checking for empty tags soon.